src/core_modules/capture-core/components/WidgetEnrollment/Actions/AddLocation/AddLocation.component.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import { IconLocation16, MenuItem } from '@dhis2/ui';
+import React, { useState, useMemo } from 'react';
+import i18n from '@dhis2/d2-i18n';
+import { dataElementTypes } from '../../../../metaData';
+import { MapCoordinatesModal } from '../../../../components/MapCoordinates';
+import { useProgramFromIndexedDB } from '../../../../utils/cachedDataHooks/useProgramFromIndexedDB';
```

Comment:
Hey,
In order to maintain the `WidgetEnrollment` self-contained we should avoid using anything from `IndexedDB`.To access the program's data, you can either pass the data down from the parent or call the [useProgram](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/WidgetEnrollment/hooks/useProgram.js) hook. This ensures that the widget still works in other apps that don't rely on the cache like the capture app. 
Thanks!